styles/styles.css

```diff
@@ -582,7 +582,7 @@ main .block.centered {
   position: relative;
 }
 
-main .default-content-wrapper {
+main .section:not(.narrow, .narrower) .default-content-wrapper {
```

Comment:
Shouldn't rather `.narrow, .narrower` apply their own padding hence overriding this one? That would make the selector more explicit.

Reply:
The problem is that padding gets added to the section-container by the narrow and narrower sections, and this applies it to the "block" wrapper

But while looking at this I found out `small-narrow` exists so there's at least one section I missed for this, and I'm sure this could and will happen again in the future, so I'll refactor it somehow to not have to add every section here